### PR TITLE
Update policy docs for /policies /policy_groups split

### PR DESCRIPTION
**Pull request by danielsdeleo**
Fixes https://github.com/chef/chef-docs/issues/563

cc @chef/lob

### CODE REVIEW

File: src/oc_erchef/raml-docs/examples/policy.json [hunk 1]
```diff
@@ -0,0 +1,485 @@
...
+      "identifier": "b611c47a7592a4abf1e7155eeb87adebcd9130fc",
+      "dotted_decimal_identifier": "51247981327127204.48398195815803783.191228277764348",
+      "cache_key": "mariadb-0.3.0-supermarket.chef.io",
+      "origin": "https://supermarket.chef.io/api/v1/cookbooks/mariadb/versions/0.3.0/download",
+      "source_options": {
+        "artifactserver": "https://supermarket.chef.io/api/v1/cookbooks/mariadb/versions/0.3.0/download",
+        "version": "0.3.0"
+      }
+    }
+  },
```
Comment (tylercloke) [inline src/oc_erchef/raml-docs/examples/policy.json]: Do you think it'd be more readable if we paired the example output down a bit?

File: src/oc_erchef/raml-docs/policy_groups.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    and policy group in order for the request to be authorized. If the
+    requestor does not have access on both objects for the requested action,
+    the server will return a 403 response.
+  uriParameters:
+    policy_group:
+      displayName: policy group
+      type: string
+    policy_name:
+      displayName: policy name
+      type: string
```
Comment (tylercloke) [inline src/oc_erchef/raml-docs/policy_groups.yml]: A short `description:` would be nice for these two parameters.

File: src/oc_erchef/raml-docs/policy_groups.yml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  get:
+    is: [ versioned ]
+    description: Return the policy document for the given policy group and policy name
```
Comment (tylercloke) [inline src/oc_erchef/raml-docs/policy_groups.yml]: Trying to end all descriptions with a `.`.

**Comment by tylercloke** [comment]
Generally LGTM other than a few notes. Document (allthethings)!

File: src/oc_erchef/raml-docs/policy_groups.yml (hunk 2 above)
Comment (danielsdeleo) [inline src/oc_erchef/raml-docs/policy_groups.yml]: I was gonna, but I can't think of a description that would actually be useful.

**Comment by pburkholder** [comment]
:heart:

**Pull request merged**